lib/voximplant_api/client.rb

```diff
@@ -51,6 +54,15 @@ def method_missing(name, *args)
 
     protected
 
+    def build_token
+      ts = Time.now.to_i
+      payload = { iss: account_id, iat: ts - 5, exp: ts + 64 }
```

Comment:
Please provide more info on why we need "issued at" field in past

Reply:
I have just copied behaviour for their own [python client](https://github.com/voximplant/apiclient-python/blob/master/src/voximplant/apiclient/client.py#L43)